frontends/election-manager/src/screens/smartcards_screen.tsx

```diff
@@ -1,31 +1,62 @@
 import React from 'react';
 import styled from 'styled-components';
+import { assert } from '@votingworks/utils';
 import { LinkButton, Prose } from '@votingworks/ui';
+import { useParams } from 'react-router-dom';
 
 import { NavigationScreen } from '../components/navigation_screen';
 import { routerPaths } from '../router_paths';
+import { SmartcardsScreenProps } from '../config/types';
 
 const Body = styled(Prose)`
   flex-grow: 1;
 `;
 
-const SuperAdminCardsLinkButton = styled(LinkButton)`
+const ToggleSmartcardTypeButton = styled(LinkButton)`
   align-self: start;
 `;
 
-export function ElectionSmartcardsScreen(): JSX.Element {
+type SmartcardType = 'election' | 'super-admin';
+
+const smartcardTypeToReadableString: Record<SmartcardType, string> = {
+  election: 'Election',
+  'super-admin': 'Super Admin',
+};
+
+function getOtherSmartcardType(smartcardType: SmartcardType): SmartcardType {
+  return smartcardType === 'election' ? 'super-admin' : 'election';
+}
+
+export function SmartcardsScreen(): JSX.Element {
+  const { smartcardType } = useParams<SmartcardsScreenProps>();
+  assert(smartcardType === 'election' || smartcardType === 'super-admin');
```

Comment:
could this be guarded by specifying it in the type of `smartcardType` in `SmartcardScreenProps`?

Reply:
Ok, the assertion seems fine then!